src/App.js

```diff
@@ -1,43 +1,25 @@
 import "./App.css";
 import logo from "./logo.png";
-
+import {Routes, Route} from "react-router-dom";
+import {Navbar} from "./components/Navbar/Navbar";
+import {Cart} from "./pages/Cart/Cart";
+import {Homepage} from "./pages/Homepage/Homepage";
+import {Login} from "./pages/Login/Login";
+import {SignUp} from "./pages/SignUp/SignUp";
+import {Wishlist} from "./pages/Wishlist/WishList";
```

Comment:
You can create an `index.js` file in the `pages` folder and export all the pages there. That way you can import them in one line

Example:

In `index.js` of pages folder:

export { Homepage } from "./Homepage/Homepage"
export { Login } from "./Login/Login"

And in `App.js`:

import { Homepage, Login,....and so on } from "./pages"


You can do the same for `components` folder in future when you have more components

Reply:
Great Idea. I'll try to implement it